app/controllers/filepaths_controller.rb

```diff
@@ -34,6 +34,11 @@ def commits
     render_json_for_api Filepath.find(params[:filepath_id]).commits
   end
 
+  #GET /api/filepaths/4/vulnerabilities
+  def vulnerabilities
+    render_json_for_api Filepath.find(params[:filepath_id]).vulnerabilities(params[:filepath_id])
```

Comment:
You don't need to pass the ID into `vulnerabilities` - just call `id` from the vulnerabilities model.
```suggestion
    render_json_for_api Filepath.find(params[:filepath_id]).vulnerabilities
```